cmds/stellar/multisig.go

```diff
@@ -31,7 +31,7 @@ func create(c *cli.Context) error {
 		return err
 	}
 
-	wallet, err := stellar.New(seed, network, nil)
+	wallet, err := stellar.New(seed, network, nil, "")
```

Comment:
Why this is passed as empty string. I assume once the horizon url is provided in cmdline it must be used in all places. and not sometimes use it and others not

Reply:
this multisig command line tool is not used by anyone